environment/__init__.py

```diff
@@ -16,3 +16,4 @@ def to_absolute_path(path):
 DEFAULT_IMAGELIST_ONE = DEFAULT_IMAGELIST[:1]
 DEFAULT_IMAGELIST_TWO = DEFAULT_IMAGELIST[:2]
 DEFAULT_IMAGELIST_THREE = DEFAULT_IMAGELIST[:3]
+PROFILE_IMAGE = DEFAULT_IMAGELIST[3:]
```

Comment:
Use a path just for profile image. Make sure to use relative path only and resolve it later in init

Reply:
Done